fastlane/metadata/android/en-US/title.txt

```diff
@@ -0,0 +1 @@
+AELF
```

Comment:
Even though the app is not available in English, maybe we can use a name equivalent to French like "AELF - Bible and day's reading" ?

Reply:
done ;)